bpfd/src/server/bpf.rs

```diff
@@ -22,7 +22,10 @@ use crate::server::{
 };
 
 // Default is Pass and DispatcherReturn
-const DEFAULT_ACTIONS_MAP: u32 = 1 << 2 | 1 << 31;
+const DEFAULT_XDP_PROCEED_ON_PASS: i32 = 2;
+const DEFAULT_XDP_PROCEED_ON_DISPATCHER_RETURN: i32 = 31;
```

Comment:
why do we need it to be an int here if we're just casting to u32 again on the next line?

Reply:
Oh I guess you did this to reuse it when adding to default below